src/main/java/com/google/sps/data/Mentor.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Mentor extends UserAccount {
+
+  public static final String ENTITY_TYPE = "Mentor";
+
+  private String dateOfBirth;
+  private String focus;
```

Comment:
This names feels a bit too generic.

Reply:
We decided to implement this as an enum instead